learn/runners/index.md

```diff
@@ -1,14 +1,26 @@
 ---
 layout: default
 title: "Beam Runners"
-permalink: /learn/sdks/
+permalink: /learn/runners/
 ---
 # Apache Beam Runners
 
-#### [Direct Runner]({{ site.baseurl }}/learn/runners/direct/) 
+A Beam Runner runs a Beam pipeline on a specific (often distributed) data processing system.
+
+## Current Runners
```

Comment:
(Also, merged both this page and the useless sdk/index.md into the new documentation/index.md, since the new layout doesn't leave a good way to make those pages visible...plus they duplicated content.)